src/AppBundle/Resources/data/posts/2016-02-10.yml

```diff
@@ -0,0 +1,41 @@
+name: "first-1-3-release-candidate"
+parent: "/news"
+title: "First Symfony CMF 1.3 Release Candidate Available"
+label: false
+format: markdown
+template: cms/news_detail.hml.twig
+publish_start_date: "2016-02-10"
+body: |
+    The release of Symfony CMF 1.3 is coming closer with the availability of
+    release candidates for all packages. Stable releases can be expected in the
+    coming weeks.
+
+    After the 1.2 release, the activity of the Symfony CMF project has dropped,
+    resulting in no new release in 2014. With this new release, the CMF is back
+    in running with full Symfony 2.8 and PHP 7 support.
+
+    ## Important new features
```

Comment:
I think it would be good to be explicit -- I would certainly be interested in seeing which packages have RC's as I don't really know. I would guess that it is the packages that are documented and doesn't include things like Resource etc.

```
- Routing 1.3 / RoutingBundle 1.3
- RoutingAuto 1.1 / RoutingAutoBundle 1.1
- MenuBundle 1.2
```